server/runtime/src/main/server/bin/service.bat

```diff
@@ -0,0 +1,551 @@
+@echo off
+
+REM -------------------------------------------------------------------------
+REM  Red Hat Data Grid Service Script for Windows
+REM    It has to reside in %JDG_HOME%\bin
+REM    It is expecting that prunsrv.exe reside in one of:
+REM      %JDG_HOME%\bin
+REM      %JDG_HOME%\..\..\jbcs-jsvc-*\sbin
+REM     example command for installation:
+REM     service.bat install /jdguser admin /jdgpass admin 
+REM      /controller localhost:11222 /shutdown-file-name shutdown-command.txt 
+REM      /node-name doc /name JDG82 /desc "JDG 82" /loglevel DEBUG
+REM
+REM  v1	2021-06-11 initial edit
+REM
+REM Author: Lei Yu
+REM ========================================================
+setlocal EnableExtensions EnableDelayedExpansion
+
+set DEBUG=0
+if "%DEBUG%" == "1" (
+	echo "Debug info enabled"
+	echo on
+)
+
+set "DIRNAME=%~dp0%"
+if "%DEBUG%" == "1" (
+	echo DIRNAME "%DIRNAME%x"
+)
+
+set TEMP_JDG_HOME=%DIRNAME%..\
+
+pushd %TEMP_JDG_HOME%
+set "RESOLVED_JDG_HOME=%CD%"
+popd
+
+rem echo %RESOLVED_JDG_HOME%
+
+set DIRNAME=
+
+if "x%JDG_HOME%" == "x" (
```

Comment:
Same comment here about JDG_HOME. Can it change to SERVER_HOME for the purposes of this script?